submissions/Oleksii_Drohachov/OOP-exercise/index.js

```diff
@@ -0,0 +1,97 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  friendsToString() {
+    return this.friends.join(", ");
+  }
+
+  output() {
+    return `${this.name}; ${this.species}; ${this.gender}; ${this.saying}; ${this.friendsToString()}`;
```

Comment:
More than 2 items. `Array#join` is still a thing.

Won't be a really good approach, but think of how destructuring could help you in more complicated cases:
`const { propA, propC, propB } = this`